### PR TITLE
Add Dockerfile for building Debian package

### PR DESCRIPTION
**Pull request by nvllsvm**
Resolves https://github.com/jellyfin/jellyfin/issues/17

### CODE REVIEW

File: Dockerfile.debian_package [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM debian:9
+
+# https://dotnet.microsoft.com/download/linux-package-manager/debian9/sdk-current
+RUN apt-get update \
+ && apt-get install -y apt-transport-https debhelper gnupg wget \
+ && wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.asc.gpg \
+ && mv microsoft.asc.gpg /etc/apt/trusted.gpg.d/ \
+ && wget -q https://packages.microsoft.com/config/debian/9/prod.list \
```
Comment (JustAMan) [inline Dockerfile.debian_package]: cannot you do it in one line like `wget -q http://url -O /etc/apt/..../microsoft-prod.list` ?.. why shuffle in two steps?

File: debian/rules [hunk 2]
```diff
@@ -14,7 +14,6 @@ override_dh_auto_test:
 override_dh_clistrip:
 
 override_dh_auto_build:
-	git submodule update --init
```
Comment (JustAMan) [inline debian/rules]: this might still be needed if we ship `taglib` as submodule (or anything else for that matter), why drop?

Comment (z3ntu) [reply debian/rules]: Afaik normally Debian packages get built from a directory where the source code is already checked out / prepared correctly, so this shouldn't be needed.

Comment (joshuaboniface) [reply debian/rules]: Yea that's how it should happen. I added this to help new users before we got the repo set up, but if we do the building its fine to remove.

File: Dockerfile.debian_package (hunk 1 above)
Comment (joshuaboniface) [reply Dockerfile.debian_package]: I personally like splitting it like this if only to make it easier to read, but no big deal.

File: README.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ The Jellyfin package is in the AUR at https://aur.archlinux.org/packages/jellyfi
 
 A package repository is available at https://repo.jellyfin.org. To use it:
 
-0. Install the `dotnet-runtime-2.1` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/dotnet-core/2.1).
+0. Install the `dotnet-runtime-2.2` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/dotnet-core/2.2).
 0. Import the GPG signing key (signed by Joshua):
```
Comment (BnMcG) [inline README.md]: Is there any reason to link the key to a specific person, or should there just be 1 Jellyfin signing key?

File: README.md [hunk 4]
```diff
@@ -75,7 +75,7 @@ A huge thanks to Carlos Hernandez who created the original Debian build configur
 
 A pre-built windows installer will be available soon. Until then it isn't too hard to install Jellyfin from Source.
```
Comment (BnMcG) [inline README.md]: Capital 'w' on Windows, lowercase 's' on source.

File: README.md [hunk 5]
```diff
@@ -75,7 +75,7 @@ A huge thanks to Carlos Hernandez who created the original Debian build configur
...
 
-0. Install the dotnet core SDK 2.1 from [Microsoft's Webpage](https://dotnet.microsoft.com/download/dotnet-core/2.1) and [install Git for Windows](https://gitforwindows.org/)
+0. Install the dotnet core SDK 2.2 from [Microsoft's Webpage](https://dotnet.microsoft.com/download/dotnet-core/2.2) and [install Git for Windows](https://gitforwindows.org/)
```
Comment (BnMcG) [inline README.md]: lowercase 'w' on web page, 'webpage' should be two words.

File: Dockerfile.debian_package (hunk 1 above)
Comment (nvllsvm) [reply Dockerfile.debian_package]: I copied/pasted from the commented link as per Micrsoft's instructions.

File: README.md (hunk 3 above)
Comment (nvllsvm) [reply README.md]: Open an issue for that.

File: README.md (hunk 4 above)
Comment (nvllsvm) [reply README.md]: Separate PR / issue please.

File: README.md (hunk 5 above)
Comment (nvllsvm) [reply README.md]: Separate PR / issue please.

File: README.md (hunk 3 above)
Comment (joshuaboniface) [reply README.md]: I set the repo up as quick as I could, so signing it as myself was the path of lease resistance. Something to revisit once we stabilize a bit more.

Comment (JustAMan) [reply README.md]: Let's tag this in separate issue for tracking - #150

**Pull request merged**